sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -445,8 +426,9 @@ const Lottery = (props: { listing: Listing }) => {
               <Button
                 variant="primary"
                 onClick={() => {
-                  // export lottery
+                  const res = onExport()
```

Comment:
Curious why we don't await this

Reply:
tbh I copied the logic from the application csv export `partners/src/pages/listings/[id]/applications/index.ts` ~line 203

where we don't await the export call. Defo down to change it to an async await though if you think it'd be better